### ISSUE TITLE
Console command 'MailAccountFetch' is failed - but isn't!

### ISSUE THREAD

**Issue by hildeb** [znuny/Znuny#269]
### Environment

- OS:          - 
- Browser:    -
- Znuny version: 6.3.4

### Expected behaviour

If a mail account fetch does not fail, there should be no error. 

### Actual behaviour

An error mail was sent by the daemon.

```
Date: Thu, 28 Jul 2022 10:10:16 +0200
From: OTRS Benachrichtigung <otrs@charite.de>
To: root@localhost
Subject: OTRS Scheduler Daemon Cron: MailAccountFetch

Console command 'MailAccountFetch' is failed.

```

Alas, I wondered what this might be about and looked at the log:

IMAGE REMOVED FOR PRIVACY REASONS!

As you can see, an error at 10:10:16.  It seems to affect TicketID=406492. But it doesn't say what the actual error is!

So I looked at the "Kommunikationsprotokoll", just to find **NO PROBLEMS** during the last hour. 10:10:16 is about 15 minutes ago:

![keine_probleme](https://user-images.githubusercontent.com/2426330/181460706-713970ff-8569-4227-8362-d89b8e441911.png)

I looked at the details for the communication at about 10:10:16 and found that mb-sicherheit was being polled (IMAPS), which goes into the "it-sicherheit" queue, so it looks like everything worked as it should. I chekced the resulting ticket: It looks ok.

### How to reproduce
?

**Comment by rkaldung** [comment]
@hildeb 1st I have removed the screenshot, there were personal e-mail addresses. I'll check which reason are possible for this result and get back to you.

**Comment by hanneshal** [comment]
@hildeb 
Some questions:
- Does it happen more often?
- What type of mail setup is used? (IMAP/POP/Postfix...) 
- Special Authentication (OAuth?)

Just as a note: if the we somehow loose the connection "to early" the process will return a non zero exit code, but the all mails may have been fetched. So this can happen in rare cases.

**Comment by hanneshal** [comment]
as a reference:
this may be fixed in 6.4.1, was internal issue, similar behaviour description. 

https://github.com/znuny/Znuny/commit/be95460afbd94d3eb6058c433c01d03322cb3303

**Comment by hildeb** [comment]
> Does it happen more often?

It happened another time today, but usually it happens every second day. But: Never happened with 6.2.x

> What type of mail setup is used? (IMAP/POP/Postfix...)

We're using both IMAPS and direct injection via **"|/opt/otrs/bin/otrs.Console.pl Maint::PostMaster::Read --untrusted"**
But this error only affects IMAPS fetching from Exchange mailboxes

**Comment by hildeb** [comment]
> as a reference: this may be fixed in 6.4.1, was internal issue, similar behaviour description.
> 
> [be95460](https://github.com/znuny/Znuny/commit/be95460afbd94d3eb6058c433c01d03322cb3303)

Does this have any dependencies or can I simply apply the diff and see what happens?

**Comment by hanneshal** [comment]
Ok, then the fix in 6.4. might help
You could try using the latest: Kernel/System/Console/Command/Maint/PostMaster/MailAccountFetch.pm

Download it and copy it to: Custom/Kernel/System/Console/Command/Maint/PostMaster/MailAccountFetch.pm

**Comment by hildeb** [comment]
Testing it

**Comment by hanneshal** [comment]
Hello @hildeb 

did the error occur again after you changed the files?

Regards
Johannes

**Comment by hildeb** [comment]
No more errors ever since!

**Comment by hanneshal** [comment]
Perfekt. Thanks for the feedback. 
I close this issue, because the fix is already included in the 6.4 release
